app/models/beer.rb

```diff
@@ -24,4 +24,12 @@ def self.search_message
       "No Matching Beer"
     end
   end
+
+  def average_rating
+    if reviews.average(:rating) == nil
+      reviews.average(:rating)
```

Comment:
What's going on here? if there are no review averages then return that? I'm not clear on what this is doing.